pay-api/src/pay_api/services/receipt.py

```diff
@@ -94,13 +99,23 @@ def get_receipt_details(filing_data, invoice_identifier, skip_auth_check):
         is_pending_invoice = (
             invoice_data.payment_method_code
             in (PaymentMethod.PAD.value, PaymentMethod.EJV.value, PaymentMethod.EFT.value, PaymentMethod.INTERNAL.value)
-            and invoice_data.invoice_status_code != InvoiceStatus.PAID.value
+            and invoice_data.invoice_status_code not in InvoiceStatus.paid_statuses()
         )
         if not is_pending_invoice and not invoice_data.receipts:
             raise BusinessException(Error.INVALID_REQUEST)
 
         invoice_reference = InvoiceReference.find_completed_reference_by_invoice_id(invoice_data.id)
 
+        applied_credits = AppliedCreditsModel.get_applied_credits_for_invoice(invoice_identifier)
+        if applied_credits:
+            receipt_details["appliedCredits"] = sum(
+                (float(credit.amount_applied) or 0.00) for credit in applied_credits
+            )
+
+        if filing_data.get("isRefund"):
```

Comment:
should probably make this named differently than `filing_data`, we're not just doing entities work